.github/aggregate.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
```

Comment:
So that's the thing: I'm purposefully keeping this script as simple as possible, with no imports, and no transpilation, no node version dependencies. It's closer to a bash script, doing a few data transformations, than a js project.